packages/next/taskfile.js

```diff
@@ -18,6 +21,18 @@ const babelOpts = {
   ]
 }
 
+export async function nccunistore (task, opts) {
+  await task
+    .source(opts.src || relative(__dirname, require.resolve('unistore')))
+    .ncc()
+    .target('dist/compiled/unistore')
+  writePackageManifest('unistore')
```

Comment:
How come you don't just do this by pushing a file in the plugin?
```
this._.files.push({ dir, base, data })
```

Reply:
I went ahead and changed it.

Reply:
I went ahead and changed it.